### PR TITLE
dTWAP and dLIMIT by orbs

### PR DESCRIPTION
**Pull request by LufyCZ**
## PR-Codex overview
This PR focuses on updating edge configurations for DCA and limit orders, implementing maintenance messages, and refining swap loading components.

### Detailed summary
- Updated edge configs for DCA and limit orders
- Added maintenance messages for DCA and limit orders
- Improved swap loading components

> The following files were skipped due to too many changes: `apps/web/src/app/(evm)/(trade)/limit/loading.tsx`, `packages/ui/src/components/settings/index.tsx`, `apps/web/src/ui/swap/swap-mode-buttons.tsx`, `apps/web/src/ui/swap/twap/twap.tsx`, `pnpm-lock.yaml`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: apps/web/src/app/(evm)/(trade)/limit/providers.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { EdgeProvider } from 'src/providers/edge-config-provider'
+import { DerivedstateSimpleSwapProvider } from 'src/ui/swap/simple/derivedstate-simple-swap-provider'
+import { getLimitEdgeConfig } from './get-limit-edge-config'
+
+export async function Providers({ children }: { children: React.ReactNode }) {
+  const config = await getLimitEdgeConfig()
```
Comment (LufyCZ) [inline apps/web/src/app/(evm)/(trade)/limit/providers.tsx:6]: miiight not revalidate?

File: apps/web/src/app/(evm)/api/config/limit/route.ts
Comment (LufyCZ) [inline apps/web/src/app/(evm)/api/config/limit/route.ts:1]: shold probably run on the edge network, if we're already using edge-config

File: apps/web/src/ui/swap/swap-mode-buttons.tsx [hunk 2]
```diff
@@ -11,31 +11,60 @@ import {
 } from '@sushiswap/ui'
 import { ShuffleIcon } from '@sushiswap/ui/icons/ShuffleIcon'
 import Link from 'next/link'
-
+// import { useCallback, useEffect, useState } from 'react'
+// import { useChainId } from 'wagmi'
 import { PathnameButton } from '../pathname-button'
 
+// const useIsTwapSupported = () => {
```
Comment (LufyCZ) [inline apps/web/src/ui/swap/swap-mode-buttons.tsx:18]: maybe remove altogether?

File: apps/web/src/ui/swap/twap/twap.tsx [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+      hidePinnedTokens={false}
+      hideSearch={false}
+    >
+      {children}
+    </TokenSelector>
+  )
+}
+
+const useTrade = () => {
+  const { data: trade, isInitialLoading } = useSimpleSwapTrade()
```
Comment (LufyCZ) [inline apps/web/src/ui/swap/twap/twap.tsx:105]: `isInitialLoading` has been removed in react-query

File: apps/web/src/ui/swap/twap/twap.tsx [hunk 4]
```diff
@@ -0,0 +1,305 @@
...
+    return `https://cdn.sushi.com/image/upload/${params.join(
+      ',',
+    )}/d_unknown.png/${src}`
+  } catch (_error) {
+    return ''
+  }
+}
+
+const useTokenList = () => {
+  const { data: customTokenMap } = useCustomTokens()
```
Comment (LufyCZ) [inline apps/web/src/ui/swap/twap/twap.tsx:142]: right when we're killing local token lists... :/

File: config/nextjs/index.js [hunk 5]
```diff
@@ -46,7 +46,7 @@ const defaultNextConfig = {
     if (config.plugins) {
       config.plugins.push(
         new webpack.IgnorePlugin({
-          resourceRegExp: /^(lokijs|pino-pretty|encoding)$/,
+          resourceRegExp: /^(lokijs|pino-pretty|encoding|keyv)$/,
```
Comment (LufyCZ) [inline config/nextjs/index.js:49]: not necessary, a relic

File: apps/web/src/ui/swap/twap/twap.tsx (hunk 4 above)
Comment (matthewlilley) [reply apps/web/src/ui/swap/twap/twap.tsx:142]: is what it is, need to replace across the rest of the app anyway

File: apps/web/src/app/(evm)/(trade)/limit/providers.tsx (hunk 1 above)
Comment (matthewlilley) [reply apps/web/src/app/(evm)/(trade)/limit/providers.tsx:6]: seems to work

**Pull request merged**